boot/boot.go

```diff
@@ -42,13 +42,13 @@ const (
 )
 
 // RebootInfo contains information about how to perform a reboot if
-// required
+// required.
 type RebootInfo struct {
-	// RebootRequired is true if we need to reboot after an update
+	// RebootRequired is true if we need to reboot after an update.
 	RebootRequired bool
-	// RebootBootloader will not be nil if the bootloader has something to say on
-	// how to perform the reboot
-	RebootBootloader bootloader.RebootBootloader
+	// BootloaderOptions can be provided if the bootloader has something
+	// to say in how to perform the system reboot.
+	BootloaderOptions *bootloader.Options
```

Comment:
nitpick: maybe it should be mentioned that the options are provided so we can find the bootloader

Reply:
good point